docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -4,8 +4,68 @@ sidebar_position: 1
 
 # Creating Service Binding
 
-In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
-- Exposing binding-metadatas by default
-If it's not the case, please refer to the following section(#)
+In this section, we consider that the service you want to connect to is already
+exposing binding metadata by either:
 
+1. Provisioned Service
+2. Direct Secret Reference
+3. Secret Generated  through CRD/CR annotations
+4. Secret Generated through OLM Descriptors
+5. Detect Bindings through owned resources
+
+You also required the application ready before creating the ServiceBinding
+resource.
+
+There are two APIs supported by the Service Binding Operator.  Both has
+`ServiceBinding` as the same name for the kind.  But the API group is different.
+
+1. binding.operators.coreos.com
```

Comment:
Which one should I use?
We should explain why we have the two apis and the impact?